### ISSUE TITLE
`postcss.vendor.unprefixed` outputted different from input value.

### ISSUE THREAD

**Issue by gucong3000** [postcss/postcss#919]
> Negative value becomes positive value, and blur distance disappears.
> Why does it happen?

Input code
```css
box-shadow: -1px -1px 1px rgba(0, 0, 0, 0.2) inset;
```

Output code
```css
box-shadow: 1px 1px rgba(0, 0, 0, 0.2) inset;
```

Function
```javascript
postcss.vendor.unprefixed(node.value);
// postcss.vendor.unprefixed('-1px -1px 1px rgba(0, 0, 0, 0.2) inset')      //=> '1px 1px rgba(0, 0, 0, 0.2) inset'
```

https://github.com/gucong3000/postcss-unprefix/issues/1

**Comment by ai** [comment]
Fix was merged

**Comment by ai** [comment]
Released in 5.2.6
